### PR TITLE
Docker Builder grammar corrections

### PR DESCRIPTION
**Pull request by RDIL**
Just getting started for the day 😉

### CODE REVIEW

File: docs/guide/docker-builder-vm.md [hunk 1]
```diff
@@ -64,8 +64,8 @@ docker build --cache-from myrepo/foo:latest \
 ### Dockerfile as a CI environment
 
 With Docker Builder there is no need to build and push custom containers so they can be used as an environment to run CI tasks in. 
-Cirrus CI can do it for you! Just specify path to a `Dockerfile` via `dockerfile` field for you container 
-declaration in `.cirrus.yml` like this:
+Cirrus CI can do it for you! Just specify path to a `Dockerfile` with the `dockerfile` field for you container 
+declaration for the `.cirrus.yml` like this:
```
Comment (fkorotkov) [inline docs/guide/docker-builder-vm.md]: Did you mean "from the"?

Comment (RDIL) [reply docs/guide/docker-builder-vm.md]: I see what you mean.  Let me try to reword that

**Comment by fkorotkov** [comment]
Are you planning to add more stuff? Or it's ready to merge?

**Comment by RDIL** [comment]
Should be good for now, I'll add more in a different PR later if needed.

**Pull request merged**